targets/targets.json

```diff
@@ -14188,7 +14189,7 @@
             "smclk_select": "HFXT",
             "smclk_div": "DIV2",
             "adc_auto_scan": 1
-        },        
+        },
```

Comment:
Future reference (as this commit is changing just one line there) but we should remove spaces in own separate commit .